# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libtwin/non-ppc.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libtwin-0.0.2/libtwin/twin_feature.c.vanilla	2007-10-19 10:54:08.000000000 +0200
+++ libtwin-0.0.2/libtwin/twin_feature.c	2007-10-19 11:00:48.000000000 +0200
@@ -54,8 +54,6 @@
 	return altivec;
 }
 
-#else
-#define _twin_have_altivec()
 #endif /* HAVE_ALTIVEC */
 
 int twin_has_feature(unsigned int feature)
@@ -65,8 +63,10 @@
 
 void twin_feature_init(void)
 {
+#ifdef HAVE_ALTIVEC
 	if (_twin_have_altivec())
 		_twin_features |= TWIN_FEATURE_ALTIVEC;
+#endif
 
 	_twin_draw_set_features();
 }
